pom.xml

```diff
@@ -51,7 +51,7 @@
         <skipUnitTests>false</skipUnitTests>
 
         <!-- The following properties configure the embedded MongoDB instance used for tests -->
-        <embeddedMongoDbVersion>2.6.5</embeddedMongoDbVersion>
+        <embeddedMongoDbVersion>3.0.2</embeddedMongoDbVersion>
```

Comment:
I think as well, we should test only the latest versions. At least for now.
If at some point we receive many requests from the users about using specific versions we might change approach.